openreview/conference/templates/decisionProcess.js

```diff
@@ -1,4 +1,4 @@
-function (){
+function processUpdate(){
```

Comment:
This was an easy fix!

Reply:
I would change the verb in the email based on the action: posted when the decision is created and updated when the decision was updated.

Reply:
I'm realizing this change affect the author notification. If the PCs decides to release the decisions after they are posted then an email will be sent to each author if author notification is available? @celestemartinez do we want this? or should we send email only when the decision is created and no email when it is updated?

Reply:
I think we should always notify when then decision has been changed (and authors are readers) but not notify if all that is changing are the readers. Is this possible?

Reply:
I think we can do a diff between the current decision and new one and check if the change is only the readers of the note.

Reply:
You are right, we should only notify the authors when the decision value has changed.

Reply:
What if only the comment is changed?

Reply:
yes, I think we should check if at least one field of the content has changed to send an email to the authors. That means if the only change is in the readers field then no email is sent.

Reply:
I would suggest to translate this process function to python, it will be easier to make the changes.

Reply:
Okay I will get started on that now!